src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[]args) {
```

Comment:
There should not be an empty line after the class declaration. Please remove the empty line to follow the Java code style guidelines.